nix/modules/vault-secrets.nix

```diff
@@ -175,12 +175,15 @@ in
               Before=${service}.service
               BindsTo=${service}.service
               StopPropagatedFrom=${service}.service
+              After=network-online.target
```

Comment:
This actually shouldn't depend on network. It should just indefinitely block on systemd-vaultd's socket.